examples/batterystats/config.json

```diff
@@ -1,20 +1,20 @@
 {
   "type": "native",
   "devices": {
-    "nexus6p": {}
+    "j7duo2": {}
   },
   "replications": 3,
   "duration": 60000,
   "monkeyrunner_path": "/home/user/Android/Sdk/tools/bin/monkeyrunner",
-  "systrace_path": "/home/user/Android/Sdk/platform-tools/systrace/systrace.py",
-  "powerprofile_path": "android-runner/example/batterystats/Scripts/power_profile.xml",
+  "systrace_path": "python2 /home/user/Android/Sdk/platform-tools/systrace/systrace.py",
```

Comment:
I remember Systrace crashing due to being launched with the wrong python version, does this still occur when this python 2 prefix is removed? If so, I think the code should manage which version of python to launch for Systrace, instead of hard coding it inside of the config.

Reply:
This is now resolved.